content/07.pathogenesis.md

```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
 
 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis
 
-[Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
+Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
+Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
+
+The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 
+A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7]. 
+Multiple research groups drafted the genome sequence of SARS-CoV-2 from different clinical samples, including bronchoalveolar lavage fluid, blood, oral and anal swabs, and cultured isolates. 
+Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10/ggjr43]. 
+However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
+
+The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection. 
+In a recent computational bioinformatics study, Grifoni et al. predicted potential B and T cell epitopes for SARS-CoV-2, providing some clues regarding which part of the pathogen may be recognized by human immune responses [@doi:10.1016/j.chom.2020.03.002]. 
```

Comment:
There was an interesting paper published yesterday showing the induction of neutralizing antibodies against the viral spike protein receptor-binding domain in humans post SARS-COV-2 infection: https://www.biorxiv.org/content/10.1101/2020.03.21.990770v1
An interesting paper that certainly merits discussion when thinking about epitope-specific responses. I can spend some time trying to identify similarities between the predicted epitopes and observed antibodies.

Reply:
To add to this, nice report on the breadth and kinetics of immune cells mobilized in response to SARS-CoV2 infection, suggesting a concerted T cell and B cell response is needed for effective viral clearance: https://www.nature.com/articles/s41591-020-0819-2

Reply:
@rishirajgoel: I am currently writing on a neutralizing antibody section in the treatments section. You are more then welcome to participate in that!